### PR TITLE
Feature/id 273/implementation about us view

### PR DESCRIPTION
**Pull request by VillalobosGarciaGaby**
**273 / Implementation About Us View**

I got to make the About Us view, as it was made in the figma MockUp:
https://www.figma.com/files/recent?fuid=1122242462142850482
 So that the user can know the purpose of Hermes, and many other services.

**Acceptance criteria.**

- [ ] Clicking on About us redirects you to the about us screen.
- [ ] When in the about us screen the user can go back with the back button.

### CODE REVIEW

File: app/src/main/java/com/isc/hermes/AboutUs.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+import androidx.annotation.IdRes;
+import androidx.appcompat.app.AppCompatActivity;
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.SpannableString;
+import android.text.style.UnderlineSpan;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+public class AboutUs extends AppCompatActivity {
```
Comment (daniel-espinoza-79) [inline app/src/main/java/com/isc/hermes/AboutUs.java:12]: Missing class commentary

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -9,4 +9,24 @@
     <string name="version"> Current Version: 0.0.0</string>
     <string name="access_token">sk.eyJ1IjoiaGVybWVzLW1hcHMiLCJhIjoiY2xpamxmbnQxMDg2aDNybGc0YmUzcHloaCJ9.__1WydgkE41IAuYtsob0jA</string>
     <string name="web_client_id">628551362001-osi4l1rcdiqt00got5n0hvn1akoc2una.apps.googleusercontent.com</string>
-</resources>
```
Comment (daniel-espinoza-79) [inline app/src/main/res/values/strings.xml:12]: Better organize the strings in folders specific to your activity

File: app/src/main/java/com/isc/hermes/AboutUs.java (hunk 1 above)
Comment (VillalobosGarciaGaby) [reply app/src/main/java/com/isc/hermes/AboutUs.java:15]: done

**Comment by EmanuelJGalindo** [comment]
I didn't see nothing critical, it's fine for me

**Pull request merged**